keras_cv/datasets/pascal_voc/segmentation.py

```diff
@@ -117,38 +129,38 @@ def _maybe_populate_voc_color_mapping():
     return VOC_PNG_COLOR_MAPPING
 
 
-def _download_pascal_voc_2012(data_url, local_dir_path=None, override_extract=False):
-    """Fetch the original Pascal VOC 2012 from remote URL.
+def _download_data_file(
+    data_url, extracted_dir, local_dir_path=None, override_extract=False
+):
+    """Fetch the original VOC or Semantic Boundaries Dataset from remote URL.
 
     Args:
-        data_url: string, the URL for the Pascal VOC data, should be in a tar package.
+        data_url: string, the URL for the SBD data, should be in a zipped tar package.
```

Comment:
I think this should be "the URL for the SBD or VOC data", right?

(Same for the Returns: docs)

Reply:
Done.